src/javascript/_common/base/client_base.js

```diff
@@ -312,6 +312,8 @@ const ClientBase = (() => {
         return (is_from_crypto ? !is_to_crypto : is_to_crypto);
     };
 
+    const isMFAccount = (loginid) => /^MF/.test(loginid || current_loginid);
```

Comment:
please don't test the login id. we don't hardcode these. we should always rely on landing company. you can instead use isAccountOfType('financial')